minio/minio
# Update browser UI assets

## Description
This commit updates the `ui-assets.go` file that is required for the changes in the UI to be reflected on the binary.

## Motivation and Context
The ui-assets.go file wasn't generated for #8362. To make the contribution process smoother, we might want to outline that this step is required for UI changes.

## How to test this PR?
After building and running the `minio` binary [without using live reload](https://github.com/minio/minio/tree/master/browser#run-minio-browser-with-live-reload), ensure on the MinIO Browser that the password field on Login does not contain the `autocomplete="new-password"` attribute.

## Types of changes

## Checklist:
- [x] Fixes a regression (#8362)
